a2p_viewProviderProxies.py

```diff
@@ -28,7 +28,7 @@
 import traceback
 
 #---------------------------------------------------------------------------
-# Modul global vars, automaticaly managed, hands off !!
+# Modul global vars, automatically managed, hands off !!
```

Comment:
what is `Modul` here? should it be `Module` or `Modal` ?

Reply:
Module...